workspaces/repo-tools/packages/cli/src/lib/workspaces/templates/workspace/package.json.hbs

```diff
@@ -35,7 +35,7 @@
   "devDependencies": {
     "@backstage/cli": "^{{version '@backstage/cli'}}",
     "@backstage/e2e-test-utils": "^{{version '@backstage/e2e-test-utils'}}",
-    "@backstage/repo-tools": "^0.8.0",
+    "@backstage/repo-tools": "^0.10.0",
```

Comment:
```suggestion
    "@backstage/repo-tools": "^{{version '@backstage/repo-tools'}}",
```

Now that I've had more time to think about this could we use the version syntax above? Then we don't need to manage this aspect as it won't be hard coded.